app/templates/src/unplanned-maintenance.html

```diff
@@ -0,0 +1,139 @@
+
+        <div class="bar" role="banner">
+          <div class="bar__inner container">
+            <div class="bar__title venus">ONS Business Surveys</div>
+          </div>
+        </div>
+
+      </header>
+
+      <div class="page__subheader"></div>
+
+      <div class="container page__container">
+
+        <div class="grid">
+
+          <div class="grid__col col-6@m">
+            <div role="main" id="main" class="page__main">
+
+            <section>
+                <h1 class="saturn">Unplanned downtime</h1>
+                <div class="panel panel--simple panel--error panel--spacious">
+                <p>We are currently experiencing technical difficulties. Our engineers are investigating and hope to have the service back up shortly.
+                  <p>Thank you for your patience.
+                </div>
+            </section>
+
+            </div>
+          </div>
+        </div>
+
+      </div>
+    </div>
+    <footer class="footer page__footer">
+      <div class="footer-nav container" role="contentinfo">
+
+        <div class="grid container">
+
+            <div class="grid__col col-4@m">
+              <div class="footer-nav__heading">
+                <h3 class="venus">Legal information</h3>
+              </div>
+              <ul class="footer-nav list">
+                <li class="footer-nav__item">
+                  <a href="https://www.ons.gov.uk/help/termsandconditions" class="footer__link">Terms and conditions</a>
+                </li>
+              </ul>
+            </div>
+
+            <div class="grid__col col-4@m">
+              <div class="footer-nav__heading">
+                <h3 class="venus">About ONS</h3>
+              </div>
+              <ul class="footer-nav list">
+                <li class="footer-nav__item">
+                  <a href="https://www.ons.gov.uk/aboutus/whatwedo" class="footer__link">What we do</a>
+                </li>
+                <li class="footer-nav__item">
+                  <a href="https://www.ons.gov.uk/help/accessibility" class="footer__link">Accessibility</a>
+                </li>
+              </ul>
+            </div>
+
+            <div class="grid__col col-4@m">
+              <div class="footer-nav__heading">
+                <h3 class="venus">Statistics</h3>
+              </div>
+              <ul class="footer-nav list">
+                <li class="footer-nav__item">
+                  <a href="https://www.statisticsauthority.gov.uk/about-the-authority/" class="footer__link">UK Statistics Authority</a>
+                </li>
+                <li class="footer-nav__item">
+                  <a href="https://www.ons.gov.uk/releasecalendar" class="footer__link">Release calendar</a>
+                </li>
+                <li class="footer-nav__item">
+                  <a href="https://www.ons.gov.uk/news">News</a>
+                </li>
+              </ul>
+            </div>
+
+        </div>
+
+      </div>
+    </footer>
+
+  </div>
+
```

Comment:
:art: Not sure we need all the whitespace :)

Reply:
I pushed a change to fix the whitespace, the change doesn't have looked like to be there though.....